dbms/src/Flash/executeQuery.cpp

```diff
@@ -98,6 +103,37 @@ BlockIO executeDAG(IQuerySource & dag, Context & context, bool internal)
 
     return res;
 }
+
+std::optional<QueryExecutorPtr> pipelineExecute(Context & context, bool internal)
```

Comment:
Would you please refine the function names in this file? We have `queryExecute`, `pipelineExecute`, `executeQuery`, `executeDAG`, kind of a mess.

Reply:
ok, refined to `queryExecute`, `executeAsPipeline`, `executeAsBlockIO`, `doExecuteAsBlockIO`.